LICENSE

```diff
@@ -1,7 +1,14 @@
-AWS JDBC Driver for PostgreSQL and all Amazon modifications to PostgreSQL JDBC Driver
+AWS JDBC Driver for PostgreSQL
+
+This driver is based on the existing PostgreSQL JDBC Driver:
+
+    https://jdbc.postgresql.org/
+
+The Amazon driver and all Amazon modifications are licensed under the 
+same license as the PostgreSQL JDBC Driver and are
 Copyright Amazon.com Inc. or affiliates. All rights reserved.
```

Comment:
So the way I read this is: 1) That this is the Amazon driver which it is not, it is the pgjdbc driver with an Aurora connection handler, and 2) That the Amazon Driver and all Amazon modifications are Copyright Amazon, only the last part is true, the Amazon additions are copyrightable, the modifications can add your copyright, but the existing code should maintain the pgjdbc copyright(s)

Reply:
Interesting. It feels like the debate here is more about naming than licensing.

I don't view this as the pgjdbc driver, if we called it that we would be misrepresenting this as being from the PostgreSQL project. It's absolutely the Amazon Driver (or in more full from the README "the Amazon Web Services (AWS) JDBC Driver for PostgreSQL"). Would the fuller name fix issues here?

I believe the original intent is that "The Amazon driver" is referring as a shorthand to the code within the https://github.com/awslabs/aws-postgresql-jdbc/tree/main/pgjdbc/src/main/java/software/aws/rds/jdbc/postgresql package. Is your objection that the Driver.java there is primarily from pgjdbc and instead you want it to say "The Amazon 'ca'"?   [with whatever noun best fits for cluster aware handler code]

Reply:
(On naming; my background is mostly Apache project based, and we may be more trademark/name sensitive than most)

Reply:
@hyandell To be clear it has never been about licensing. 
My issue has always been attribution.
Amazon has added ~4600 lines of code to 60,000 exclusive of tests. I'm having trouble wrapping my head around how one can claim it's Amazon's driver?

Reply:
I think there are a lot of overlapping issues to discuss here.

My understanding in general Open Source philosophies is that it is fine to say that something is the same product when there are tiny tweaks. Like a Debian package for a project adding backports or tiny build tweaks but still calling it the same name. Larger changes however mean a name change, otherwise the original project suffers from bug reports that don't relate to them at all. Flashbacks to all the arguments over Debian Iceweasel. So namewise, I definitely think we (wearing my AWS OSPO hat) should **not** be calling this pgjdbc. 

On the other hand, I definitely agree that we should not be saying it is all our own work. I hope we're not saying that anywhere and will support feedback to fix any of that. That being core to why we (general permissive community) have attribution requirements in permissive licensing, to make sure that nobody passes off their project as being all their own work.

Do you have some proposed alternative language? (I don't mean that from "Please do my work for me" but just to help me understand your perspective)